ady00.md

```diff
@@ -0,0 +1,33 @@
+
+---
+name: Advay Bajpai (github - @ady00)
+project: Guitar Effects Pedal/Board
+---
+
+#  Project Name
+
+##  Summary
+
+Hey there! For my project, I'd like to construct a guitar effects pedal using a Raspberry Pi. As an avid guitarist and programmer, being able to merge two of my favorite hobbies would be amazing. In summary, the device I'm planning on building will be a multi-effect guitar pedal, complete with distortion, echo, reverb, and really just about any effect as long as it's programmable. 
+
+##  Plan
+
+I'll be utilizing a Raspberry Pi 4B as the brains of the operation to run effects plugins. To convert analog signals from my guitar into digital signals the Raspi can process, I'll use a HiFiBerry ADC + DAC, that will sit atop the Raspberry Pi. The Raspi will be running on [Elk Audio OS](https://github.com/elk-audio/elk-pi/releases/tag/0.9.0), an open source operating system with low latency for audio-based projects. And finally, I'll be running a real-time VST3 plugin, NeuralPi, which is a series of Neural Networks designed to emulate famous guitar amps and effects. I'll be following [this tutorial](https://medium.com/nerd-for-tech/neural-networks-for-real-time-audio-introduction-ed5d575dc341) on the applications of AI for real time audio as I go. Thanks for checking out my proposal! 
+
+##  Budget
+
+| Product  | Supplier/Link  | Cost  |
+| --------------- | ------------------------------------- | ------ |
+| Raspberry Pi 4B  | [Adafruit](https://www.adafruit.com/product/4292) | $45.00 + $30 shipping |
```

Comment:
This appears to be out of stock. Any chance you'd be able to find a vendor that has such devices available? This may be helpful: https://rpilocator.com/